### DISCUSSION TITLE
v0.11.0

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#320] [Releases]
## What's Changed
* User management by @UnAfraid in https://github.com/alexjustesen/speedtest-tracker/pull/306
* select a manually entered server ID by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/315
* Unhardcoding cd-docker-hub.yml by @UnAfraid in https://github.com/alexjustesen/speedtest-tracker/pull/316
* Telegram notification channel by @garanda21 in https://github.com/alexjustesen/speedtest-tracker/pull/265
* Failed speedtests by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/318
* Release v0.11.0 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/319

## New Contributors
* @garanda21 made their first contribution in https://github.com/alexjustesen/speedtest-tracker/pull/265

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.10.0...v0.11.0

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.11.0'>v0.11.0</a>.</em>

**Comment by Kofl** [comment]
Awesome release, thanks a lot for taking over the project!
